File: _src/chapter9/memcache_example.php
<?php

$memcache = new Memcache();
$memcache->connect("127.0.0.1", 5701)
  or die("Could not connect to Hazelcast");

$city = Array();
$city["name"] = "London";
$city["country"] = "GB";
$city["population"] = 8174100;

$memcache->set("London-GB", $city)
  or die("Failed to save city");

$london = $memcache->get("London-GB");

echo var_export($london, true) ."\n";

?>

File: _src/chapter9/memcache_example.py
#!/usr/bin/env python

import memcache
mc = memcache.Client(['127.0.0.1:5701'], debug=0)

city = {
  "name": "London",
  "country": "GB",
  "population": 8174100
}

mc.set("London-GB", city)
london = mc.get("London-GB")

print london
